MetaBrainz.MusicBrainz.Tests/BrowseTests.cs

```diff
@@ -0,0 +1,76 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+
+using FluentAssertions;
+
+using Xunit;
+
+namespace MetaBrainz.MusicBrainz.Tests {
+  public class BrowseTests : MusicBrainzTest {
+
+    private const string ReleaseTestData = @"{
+   ""releases"":[
```

Comment:
I might be more inclined to use a .json file stored as embedded resource; the overhead of loading that is small enough, and it makes the test data easier to work with.

Reply:
Agreed - I was thinking this would be a better path to go down but kept the POC simple.